### ISSUE TITLE
[Qt Quick][Windows] addIgnoreObject does not work on child windows

### ISSUE THREAD

**Issue by midnightprioriem** [wangwenx190/framelesshelper#45]
The ```addIgnoreObject``` function does not work for child windows created in QML. Controls within the title bar space are not able to be interacted with in child windows.

CustomWindow.qml:
```qml
Window {
    id: window
    visible: true
    width: 800
    height: 600
    title: qsTr("Hello, World!")
    color: "#803b3642"

    FramelessHelper {
        id: framelessHelper
    }

    Rectangle
    {
        anchors.fill: parent
        color: "transparent"
        border.width: .5
        border.color: "#3b3642"
    }

    Row {
        anchors.top: parent.top
        anchors.right: parent.right

        MinimizeButton {
            id: minimizeButton
            onClicked: window.showMinimized()
            Component.onCompleted: framelessHelper.addIgnoreObject(
                                       minimizeButton)
        }

        MaximizeButton {
            id: maximizeButton
            maximized: window.visibility === Window.Maximized
            onClicked: {
                if (maximized) {
                    window.showNormal()
                } else {
                    window.showMaximized()
                }
            }
            Component.onCompleted: framelessHelper.addIgnoreObject(
                                       maximizeButton)
        }

        CloseButton {
            id: closeButton
            onClicked: window.close()
            Component.onCompleted: framelessHelper.addIgnoreObject(
                                       closeButton)
        }
    }


    Component.onCompleted: {
        framelessHelper.setWindowFrameVisible(false)
        framelessHelper.removeWindowFrame(true)
        framelessHelper.setBlurEffectEnabled(true)//,true, "#803b3642")
    }


}
```

main.qml:
```qml
CustomWindow {

    Button
    {
        anchors.centerIn: parent
        onClicked: {childWindow.show();}
    }

    CustomWindow
    {
        id: childWindow
        visible: false
    }

}
```

Result:

https://user-images.githubusercontent.com/6120081/109992847-ab61e880-7cd9-11eb-9495-2910cd179013.mp4

**Comment by midnightprioriem** [comment]
This issue should be fixed with #46

**Comment by wangwenx190** [comment]
> This issue should be fixed with #46

Thanks for the quick fix!
